asf_heat_pump_suitability/pipeline/run_scripts/add_features.py

```diff
@@ -96,34 +96,30 @@ def main(epc_path: str, save_output: Optional[str] = None) -> pl.DataFrame:
     logging.info("Adding off gas grid column to EPC")
     off_gas_postcodes = off_gas.process_off_gas_data()
     enhanced_epc_df = off_gas.add_off_gas_feature(enhanced_epc_df, off_gas_postcodes)
-    logging.info("Adding Historic England listed buildings to EPC")
```

Comment:
Here the logic remains the same but I shortened some of the variable names to increase readability of the code.